src/ROOTex.jl

```diff
@@ -45,6 +45,22 @@ function Fit(g::TGraph, tf1::TF1, option::String = "", goption::String = "", rxm
     Fit(g, CxxPtr(tf1), option, goption, rxmin, rxmax)
 end
 
+#---TFile extensions-------------------------------------------------------------------------------
+function GetObject(file::Union{ROOT.TDirectoryFile, CxxPtr{<:ROOT.TDirectoryFile}}, name::AbstractString)
```

Comment:
Type-stable version of the function added. An exception is now thrown by default instead of returning C_NULL. Providing a default value to be returned in place of throwing an exception is also supported.